docs/docs.md

```diff
@@ -0,0 +1,585 @@
+# `/event/{eventKey}/info`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  webcasts: {
+    type: "twitch" | "youtube"
+    url: string
+  }[]
+  location: {
+    name: string
+    lat: number
+    lon: number
+  }
+  name: string
```

Comment:
Are we going to do the shortname/fullname thing again or just go with one of them?

Reply:
I think we'll just do shortname everywhere